specification/cognitiveservices/data-plane/ImageSearch/v1.0/ImageSearch.json

```diff
@@ -0,0 +1,2171 @@
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "X-MSEdge-ClientIP",
+            "x-ms-client-name": "ClientIp",
+            "in": "header",
+            "description": "The IPv4 or IPv6 address of the client device. The IP address is used to discover the user's location. Bing uses the location information to determine safe search behavior. Although optional, you are encouraged to always specify this header and the X-Search-Location header. Do not obfuscate the address (for example, by changing the last octet to 0). Obfuscating the address results in the location not being anywhere near the device's actual location, which may result in Bing serving erroneous results.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "X-Search-Location",
+            "x-ms-client-name": "Location",
+            "in": "header",
+            "description": "A semicolon-delimited list of ke
```

Comment:
Please add descriptions to the properties.

Reply:
resolved